app/controllers/users/contact_details.js

```diff
@@ -12,15 +12,40 @@ export default Ember.Controller.extend(
     user: Ember.computed.alias("model.user"),
     userService: Ember.inject.service(),
 
-    invalidEmail: Ember.computed("user.email", function() {
-      const emailRegEx = new RegExp(regex.EMAIL_REGEX);
-      return this.get("user.email").match(emailRegEx);
-    }),
+    isValidEmail(email) {
+      return email.match(new RegExp(regex.EMAIL_REGEX));
```

Comment:
if this needs to return true/false,
then return `regex.EMAIL_REGEX.test(email)` should work and can get rid of `Boolean(this.isValidMobile(this.get("mobileNumber")))` this line in the below method